README.md

```diff
@@ -34,7 +34,12 @@ $ pip install .
 If you plan to develop the package, you may run it directly from source:
 
 ```
-$ python setup.py develop       # Do this once to add pkg to Python path
+$ pip install -e .
+```
+
+Each time after modifying a Cython or C file, also run:
```

Comment:
That's a bit counter-intuitive to me, and relies on some pip magic.  The way I think of it is: "add package to system path, then recompile extensions as you work".  But, I won't push back if you feel strongly.